README.md

```diff
@@ -1094,6 +1094,13 @@ $connector->connect('google.com:80')->then(function (React\Socket\ConnectionInte
   Internally, the `tcp://` and `tls://` connectors will always be wrapped by
   `TimeoutConnector`, unless you disable timeouts like in the above example.
 
+> Aside from defaulting to the `DnsConnector` the `Connector` also supports the 
+  `HappyEyeBallsConnector` by setting the `happy_eyeballs` option to `true`. It's 
+  set to false by default to be backwards compatible with older systems only 
+  supporting IPv4. By setting the `happy_eyeballs` option to `true` new connection 
+  attempts are made over both IPv6 and IPv4 with the priority for IPv6. And return 
+  the connection attempt that connects first.
```

Comment:
I agree that this feature makes perfect sense and agree that having a feature toggle makes sense for avoid any problems with either edge cases we didn't cover or higher-level application logic that doesn't cope well with IPv6.

I don't consider this to be a BC break on our API side, so I think this should (eventually) be enabled by default. Having this disabled by default might make sense for a transition period. I personally don't mind either way, but would lean towards enabling this from the beginning to gather more feedback.

What do you think about this? Can we update the text to describe this (future) default behavior?

Reply:
When we initially talked about this you suggested disabling it by default where I wanted to enable by default IIRC. But I could be wrong and am glad to hear you also rather see it enabled by default. I'll change it as such :tada: .

Also I want to do a follow up PR deprecating the DnsConnector and marking it for removal in 2.0.

Reply:
I'm not sure what was discussed before, but I think you're on the right track with your suggestion :+1: 

Deprecating `DnsConnector` sounds reasonable. Make sure said follow-up PR discusses what happens when `happy_eyeballs` will be turned off for whatever reasons and the application only wants to support IPv4.

Reply:
> I'm not sure what was discussed before, but I think you're on the right track with your suggestion 

It's been a while :).

> Deprecating `DnsConnector` sounds reasonable. Make sure said follow-up PR discusses what happens when `happy_eyeballs` will be turned off for whatever reasons and the application only wants to support IPv4.

Will do